app/javascript/flavours/glitch/styles/components/drawer.scss

```diff
@@ -86,9 +86,8 @@
     box-sizing: border-box;
     margin: 0;
     border: none;
-    padding: 10px 30px 10px 10px;
+    padding: 15px 30px 15px 15px;
```

Comment:
Not really sure how it looks in the single column version but in mastofe with the multi columns it looks quite off as the search icon goes uncentered.

Reply:
The part with the search icon has diverged from upstream and is pretty weird… I will take care of that eventually.